Rubberduck.Core/UI/Command/ComCommands/ComCommandBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using NLog;
+using Rubberduck.VBEditor.Events;
+
+namespace Rubberduck.UI.Command.ComCommands
+{
+    public abstract class ComCommandBase : CommandBase
+    {
+        private readonly IVbeEvents _vbeEvents;
+
+        protected ComCommandBase(ILogger logger, IVbeEvents vbeEvents) : base(logger)
+        {
+            _vbeEvents = vbeEvents;
+        }
+        
+        protected override bool EvaluateCanExecute(object parameter)
+        {
+            return !_vbeEvents.Terminated && base.EvaluateCanExecute(parameter);
+        }
```

Comment:
This will not work as intended because subclasses override this method.

Btw, this implementation is equivalent to `protected override bool EvaluateCanExecute(object parameter) => !_vbeEvents.Terminated`.